src/components/internal/Card/Card.tsx

```diff
@@ -31,6 +39,35 @@ const Card = ({
     testId,
 }: PropsWithChildren<CardProps>) => {
     const Tag = el || 'header';
+    const [showContent, setShowContent] = useState(false);
+    const cardId = useMemo(() => uuid(), []);
+
+    const toggleExpansion = useCallback(() => {
+        if (expandable) {
+            setShowContent(!showContent);
+        }
+    }, [expandable, showContent]);
```

Comment:
Instead of having `showContent` as a dependency here, you can switch to the computed version of the state setter.

```suggestion
    const toggleExpansion = useCallback(() => {
        if (expandable) setShowContent(showContent => !showContent);
    }, [expandable]);
```